### PR TITLE
Cleanup setup/metadata files

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,16 +1,21 @@
+"""
+A full-screen image modal XBlock
+"""
 import json
 from os import path
 from setuptools import setup
 
 
+version = '0.5.0'
+description = __doc__.strip().split('\n')[0]
```
Comment (stvstnfrd) [inline setup.py]: Take the first line of the docstring.

File: setup.py [hunk 2]
```diff
@@ -1,16 +1,21 @@
+"""
+A full-screen image modal XBlock
+"""
 import json
 from os import path
 from setuptools import setup
 
 
+version = '0.5.0'
```
Comment (stvstnfrd) [inline setup.py]: This is the main thing that will be updated frequently;
let's make it easier by putting it up top.

File: package.json [hunk 3]
```diff
@@ -1,33 +1,10 @@
 {
-  "name": "xblock-image-modal",
-  "title": "Image Modal XBlock",
-  "description": "A fullscreen image modal XBlock.",
-  "version": "0.5.0",
```
[the remark this answers is not in the document]
Comment (stvstnfrd) [reply package.json]: That metadata is only needed when installing via a package manage, like `npm`.
We're not doing that; we're only using this file as a standard way of listing/installing `node` packages that we're running as command-line scripts via `npm install`.

Having the metadata stored in two places means we now have the chore of keeping the two of them in sync. It's unlikely that, nor should people have to, remember to update the version in two files; we forget to bump the version number once, enough as it is :P

No reward, so why bother?

[the remark this answers is not in the document]
Comment (caesar2164) [reply package.json]: Oh gotcha, we're making the `setup.py` the single source of truth for the xblock, sounds good.

**Pull request merged**